### PR TITLE
[Dev] Smart Search: Index custom fields

### PR DESCRIPTION
**Pull request by max123kl**
Pull Request for Issue #4

### Summary of changes
file added

### CODE REVIEW

File: J4_dev_manual/Creating a Smart Search plug-in.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+As such, one way to test is to use the command line indexer.\[2\]
+
+Open a command prompt/ssh session on the server and go to the root
+directory of the website. From there, you can run the indexer and errors
+and log messages will go to the screen. `php cli/finder_indexer.php`
+
+## Notes
+
+<references/>
```
Comment (max123kl) [inline J4_dev_manual/Creating a Smart Search plug-in.md:183]: '''suggestion
delete line

Comment (max123kl) [inline J4_dev_manual/Creating a Smart Search plug-in.md:183]:
``` suggestion

Comment (max123kl) [reply J4_dev_manual/Creating a Smart Search plug-in.md:183]: delete line

Comment (ceford) [reply J4_dev_manual/Creating a Smart Search plug-in.md:183]: How do you pick out lines for suggested changes?

**Pull request closed without merging**